recipes/hapog/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set version = "1.3.2" %}
+
+package:
+  name: hapog
+  version: {{ version }}
+
+build:
+  number: 0
+
+source:
+  url: https://github.com/institut-de-genomique/HAPO-G/archive/refs/tags/{{ version }}.tar.gz
+  sha256: "46e0d102467276d0aee5c493e5e678c76120760f0dbe57caf4d880d366dcb285"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - make
+  host:
+    - 'python>=3.7'
+    - htslib
+    - setuptools
+  run:
+    - htslib
+    - 'python>=3.7'
+    - setuptools
+    - biopython
+    - bwa
+    - samtools
+    - minimap2
+
+test:
+  commands:
+    - hapog -h
```

Comment:
Which is the command to run? The README of the [upstream tool](https://github.com/institut-de-genomique/HAPO-G#standard-pipeline) talks about running a `hapog.py` - does `hapog` here correspond to that Python file? And is `hapog_bin` a compiled module? Would the user ever run `hapog_bin` directly?

Reply:
The command to run is `hapog` as it is a wrapper around a C binary that was called `hapog_bin` here. In most cases, the user won't have to run this manually but in the case they need to, we would prefer if the binary was easily accessible.